examples/arc_ball_camera/resources/display_config.ron

```diff
@@ -1,3 +1,8 @@
+/*!
+    @import /amethyst_window/src/config.rs#DisplayConfig
+    DisplayConfig
+*/
+
 (
-  title: "Arc Ball Camera Example",
```

Comment:
btw, is removing trailing commas the default setting for the RON formatter?
I might be caring too much, but `rustfmt` has trailing commas if the type's parameters are over multiple lines
(feel free to ignore this comment, I'll conveniently forget about it next time)

Reply:
Yes, it is the default setting for now.  Thank you for the heads up about rustfmt, I'll update the plugin to have similar functionality (configurably) shortly.

Reply:
Yes, it is the default setting for now. Thank you for the heads up about rustfmt, I'll update the plugin to have similar functionality (in a configurable way) shortly.